packages/changed-elements-react/src/widgets/ChangedElementsWidget.tsx

```diff
@@ -365,6 +386,9 @@ export class ChangedElementsWidget extends Component<ChangedElementsWidgetProps,
           <WidgetComponent.Body data-testid="comparison-legend-widget-content">
             {this.state.loaded ? this.getChangedElementsContent() : this.getLoadingContent()}
           </WidgetComponent.Body>
+          <WidgetComponent.ToolBar>
+            {this.props.useV2Widget && (!!this.props.feedbackUrl) ? <FeedbackButton feedbackUrl={this.props.feedbackUrl ?? ""}></FeedbackButton> : <></>}
+          </WidgetComponent.ToolBar>
```

Comment:
When only one conditional branch is used, you can simplify the ternary expression as follows:
```tsx
{
  condition &&
  <TrueBranch />
}
```